app/mailers/user_notifier.rb

```diff
@@ -6,4 +6,10 @@ def send_signup_email(user)
     mail(to: @user.email,
          subject: "Thanks for signing up for with Hipster Bytes")
   end
+
+  def order_confirmation(order)
+    @order = order
+    # require 'pry'; binding.pry
+    mail(to: @order.user.email, subject: 'Order has been received')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.